Content/HowTo/distributions-web.md

```diff
@@ -0,0 +1,22 @@
+#Distribution's official web pages about Pinta
+
+##BSD distributions
+[FreeBSD][1]<br>
+[OpenBSD][2]
+
+##Linux distributions
+[Arch Linux][3]<br>
+[Debian][4]<br>
+[Fedora][5]<br>
+[Linux Mint][6]<br>
+[openSuse][7]<br>
```

Comment:
I think the capitalization is supposed to be openSUSE, not openSuse?.

Also, it might be better to format these using a list instead of lines separated by `<br>`